.build/googet/google-cloudops-opentelemetry-collector.goospec

```diff
@@ -1,26 +1,26 @@
 {
   "name": "google-cloudops-opentelemetry-collector",
-  "version": "${VERSION}",
+  "version": "${PKG_VERSION}",
   "arch": "${ARCH}",
   "authors": "OpenTelemetry Community",
   "license": "http://www.apache.org/licenses/LICENSE-2.0",
   "description": "Google Cloud Operations OpenTelemetry Collector based Monitoring Agent",
   "files": {
-    "bin/google-cloudops-opentelemetry-collector_windows_${GOARCH}.exe": "<ProgramFiles>/Google/Cloud Operations/OpenTelemetry Collector/google-cloudops-opentelemetry-collector.exe",
-    "config.yaml": "<ProgramFiles>/Google/Cloud Operations/OpenTelemetry Collector/config.yaml"
+    "bin/google-cloudops-opentelemetry-collector_${GOOS}_${GOARCH}.exe": "<ProgramFiles>/Google/Cloud Operations/OpenTelemetry Collector/google-cloudops-opentelemetry-collector.exe",
+    "config/config-windows.yaml": "<ProgramFiles>/Google/Cloud Operations/OpenTelemetry Collector/config.yaml"
   },
   "install": {
-    "path": "packaging/googet/install.ps1"
+    "path": ".build/googet/install.ps1"
   },
   "uninstall": {
-    "path": "packaging/googet/uninstall.ps1"
+    "path": ".build/googet/uninstall.ps1"
   },
   "sources": [{
       "include": [
-        "./bin/google-cloudops-opentelemetry-collector_windows_${GOARCH}.exe",
-        "./config.yaml",
-        "./packaging/googet/install.ps1",
-        "./packaging/googet/uninstall.ps1"
+        "bin/google-cloudops-opentelemetry-collector_${GOOS}_${GOARCH}.exe",
```

Comment:
According to https://github.com/golang/go/blob/0951939fd9e4a6bc83f23c42e8ddff02b29c997e/src/go/build/syslist.go#L10, `${GOOS}` will always resolve to `windows` for Windows, but may still be useful for naming standardisation.

Reply:
Yea I was pretty 50/50 with including that in this file, but was trying to standardise as much as possible. Jingbo is going to be adding Linux packaging to this repo very soon.